### ISSUE TITLE
A11y: All links should use blue #0070F0

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/vacant-lots-proj#332]
## Describe the task
The blue used for the links on the action page #3B82F6 does not meet 4:5:1 color contrast. The blue on the links on the about page pass contrast #0070F0.

Update styles to use this blue throughout.

https://webaim.org/resources/contrastchecker/  https://dequeuniversity.com/rules/axe/4.8/color-contrast

## Acceptance Criteria

- [ ] All blue links will use the darker blue

**Comment by liathirsh** [comment]
Hi, I'd like to take this on!
